File: SofiForce.Web/ClientApp/src/app/core/Models/SearchModels/RepresentativeJourneySearchModel.ts
import { BaseSearchModel } from "./baseSearchModel";

export interface RepresentativeJourneySearchModel extends BaseSearchModel {
    representativeId: number | null;
    represtitiveUserId: number | null;
    supervisorId: number | null;
    routeId: number | null;
    branchId: number | null;
    kindId: number | null;
    routeTypeId:number | null;

    representativeCode: string | '';
    represtitiveUserCode: string | '';
    supervisorCode: string | '';
    routeCode: string | '';
    branchCode: string | '';
    kindCode: string | '';
}

File: SofiForce.Models/Models/SearchModels/RepresentativeJourneySearchModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class RepresentativeJourneySearchModel : BaseSearchModel
    {


		public Int32? RepresentativeId { get; set; }
		public Int32? ReprestitiveUserId { get; set; }
		public Int32? SupervisorId { get; set; }
		public Int32? RouteId { get; set; }
		public Int32? BranchId { get; set; }
		public Int32? KindId { get; set; }

		public Int32? RouteTypeId { get; set; }

	}
}